multiversx_sdk_cli/transactions.py

```diff
@@ -148,12 +148,34 @@ def _send_transaction_and_wait_for_result(proxy: INetworkProvider, payload: ITra
 
 
 def tx_to_dictionary_as_inner(tx: Transaction) -> Dict[str, Any]:
-    dictionary = tx.to_dictionary()
-    dictionary["receiver"] = base64.b64encode(bytes.fromhex(tx.receiver.hex())).decode()  # type: ignore
-    dictionary["sender"] = base64.b64encode(bytes.fromhex(tx.sender.hex())).decode()  # type: ignore
-    dictionary["chainID"] = base64.b64encode(tx.chainID.encode()).decode()
-    dictionary["signature"] = base64.b64encode(bytes(bytearray(tx.signature))).decode()
-    dictionary["value"] = tx.value
+    dictionary: Dict[str, Any] = {}
```

Comment:
Perhaps we can also rename the function to `tx_to_dictionary_as_inner_for_relayed_v1`. Or perhaps can we use the builder from core instead? Can be in a next PR.

Reply:
Renamed to `tx_to_dictionary_as_inner_for_relayed_v1`. Was thinking about using the builder, but since here we simply format the data field of the relayed transaction it's not suitable. We also marked the builders as deprecated, will probably use the factory, when it's ready.